test/WebJobs.Script.Tests/Configuration/ScriptApplicationHostOptionsSetupTests.cs

```diff
@@ -13,6 +14,32 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
 {
     public class ScriptApplicationHostOptionsSetupTests
     {
+        [Theory]
+        [InlineData("1", false)]
+        [InlineData("0", false)]
+        [InlineData("true", true)]
+        [InlineData("True", true)]
+        [InlineData("false", false)]
+        public void SequentialHostRestartRequired_Configured(string value, bool expectedResult)
+        {
+            ScriptApplicationHostOptions options = new ScriptApplicationHostOptions();
+
+            IConfiguration config = new ConfigurationBuilder()
+                .AddInMemoryCollection(new Dictionary<string, string>
+                {
+                { "AzureFunctionsJobHost:SequentialRestart",  value },
```

Comment:
```suggestion
                    { "AzureFunctionsJobHost:SequentialRestart",  value },
```

Also tiniest nit, I like the newer dictionary initializer

``` csharp
{
    ["AzureFunctionsJobHost:SequentialRestart"] = value,
}
```